exec/java-exec/src/test/java/org/apache/drill/exec/server/TestDrillbitResilience.java

```diff
@@ -650,6 +650,7 @@ public void dataArrived(final QueryDataBatch result, final ConnectionThrottle th
 
   @Test // DRILL-2383: Cancellation TC 3: cancel after all result set are produced but not all are fetched
   @Repeat(count = NUM_RUNS)
+  @Ignore
```

Comment:
Please log a JIRA and reference it in this ignore statement

Reply:
@parthchandra added jira to ignore annotation https://issues.apache.org/jira/browse/DRILL-6228